src/main/resources/edu/oregonstate/mist/api/resource.properties

```diff
@@ -21,3 +21,15 @@ conflict.details = https://developer.oregonstate.edu/documentation/error-referen
 internalServerError.userMessage = The application ran into an internal error. If the problem persists, please contact application support.
 internalServerError.code = 1500
 internalServerError.details = https://developer.oregonstate.edu/documentation/error-reference#1500
+
+#Inventory API Error Messages
+inventory.invalidUUID = ID is not a valid UUID. Inventory ID must follow UUID structure detailed here: https://tools.ietf.org/html/rfc4122.html
```

Comment:
I think the invalidUUID and idExists messages are more generic user messages that are probably not specific to inventory-api.